js/about/contributionStatement.js

```diff
@@ -430,4 +455,242 @@ function longFormattedDateFromTimestamp (timestamp) {
   return `${momentDate.format('MMMM Do')} at ${momentDate.format('h:mma')}`
 }
 
+const containerMargin = '25px'
+const boxMargin = '15px'
+const summaryBoxTableBorder = '10px'
+const summaryBoxTableMargin = '10px'
+const summaryBoxTablePadding = '10px'
+const lightGray = '#f7f7f7'
+const braveLogoWidth = '50px'
+const braveLogoHeight = '50px'
+const braveLogo = require('../../app/extensions/brave/img/braveAbout.png')
+const verifiedIcon = require('../../app/extensions/brave/img/ledger/verified_green_icon.svg')
+
+const styles = StyleSheet.create({
```

Comment:
I'm not sure how worth it is. I left comments and grouped with `__` for each function, and I think it is not so big to separate into the objects.

If we will separate them, let's do so after everything is refactored and we create the rules which decide when to separate objetcs; Otherwise it will definitely lead to inconsistency and we would not like to see that.